test/es-module/test-esm-loader-threads.mjs

```diff
@@ -16,6 +16,9 @@ describe('off-thread hooks', { concurrency: true }, () => {
       fixtures.path('es-module-loaders/workers-spawned.mjs'),
     ]);
 
+    console.log(stderr);
+    console.log(stdout);
+
```

Comment:
I assume this is a leftover from your debugging?

```suggestion
```

Reply:
indeed. removed